cohort-engine/src/main/java/com/ibm/cohort/engine/BooleanEvaluationCallback.java

```diff
@@ -44,6 +45,11 @@ public void onEvaluationComplete(String contextId, String expression, Object res
 				}
 				logger.info(String.format("Expression: \"%s\", Result: %s", expression, result));
 			}
+			else {
+				throw new RuntimeException("Only boolean CQLs are currently supported!");
+			}
+		} else {
+			throw new RuntimeException("Null result is unsupported");
```

Comment:
I think these two exceptions should log the parameters like what happens on line 46.
In fact, I think it's probably a good idea to extend the log statement on line 46 to include the context id.
I guess logging `result` isn't necessary on the null case specifically.

Reply:
Fixed